app/steps/many_members_step.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ManyMembersStep < Step
+  step_attributes :members
+
+  def valid?
+    members.each do |member|
+      member.errors.clear
+      validate_household_member(member)
+    end
+
+    members.map(&:errors).all?(&:blank?)
```

Comment:
If the errors are not empty/blank I think we should bubble up the contents of those objects up to the step.